source/MaterialXRuntime/Identifiers.cpp

```diff
@@ -74,6 +74,7 @@ namespace Identifiers
     const RtIdentifier DEFAULT("default");
     const RtIdentifier DEFAULTINPUT("defaultinput");
     const RtIdentifier UNKNOWN("unknown");
+    const RtIdentifier INFO("info");
```

Comment:
The MaterialX attribute name is called "doc". (documentation). I think it's better to go with DOCUMENTATION("doc").

Reply:
Agree with Bernard to use "doc" as that is used in the MaterialX specification.

For the identifier constant we should use DOC("doc") though, since a constant in the `Identifiers` namespace is expected to be named exactly according to the string value it holds.